website/docs/components/button-set/partials/code/how-to-use.md

```diff
@@ -9,17 +9,57 @@ The basic invocation requires two or more buttons to be provided as children:
 </Hds::ButtonSet>
 ```
 
+Other button-like components can also be used as children such as the `Dropdown`.
+
+```handlebars
+<Hds::ButtonSet>
+  <Hds::Dropdown as |D|>
+    <D.ToggleButton @color="secondary" @text="Select an option" />
+    <D.Title @text="Title Text" />
+    <D.Description @text="Descriptive text goes here." />
+    <D.Interactive @href="#">Add</D.Interactive>
+    <D.Interactive @href="#">Add More</D.Interactive>
+    <D.Interactive @href="#">Add Another Thing Too</D.Interactive>
+    <D.Separator />
+    <D.Interactive @route="components" @icon="trash" @color="critical">Delete</D.Interactive>
+  </Hds::Dropdown>
+
+  <Hds::Button @text="Submit" type="submit" />
+</Hds::ButtonSet>
+```
+
 ### Equal width buttons
 
-If you want to have buttons with equal width, apply a width (via inline style or CSS class) to the `ButtonSet` container and the argument `@isFullWidth={{true}}` to the `Button` components:
+If you want buttons with equal widths, set `@isFullWidth` to `true` on the `Button` components. To constrain the overall max-width, you can also add a width to the `ButtonSet` container (via an inline style or CSS class).
+
+Note: The ButtonSet is full-width (100%) by default.
 
 ```handlebars
-<Hds::ButtonSet {{style width="15rem"}}>
+<Hds::ButtonSet {{style maxWidth="15rem"}}>
   <Hds::Button @text="Save" @isFullWidth={{true}} />
   <Hds::Button @text="Cancel" @color="secondary" @href="https://hashicorp.com" @isFullWidth={{true}} />
 </Hds::ButtonSet>
 ```
 
+Issue: The `Dropdown` does not have an `isFullWidth` option for the outer wrapper. Setting isFullWidth to “true” on the nested ToggleButton does not have the same effect.
```

Comment:
Maybe this should be a warning banner?

```suggestion
!!! Warning 

**Code alert**

The `Dropdown` does not have an `isFullWidth` option for the outer wrapper. Setting isFullWidth to “true” on the nested ToggleButton does not have the same effect.
!!!
```

Reply:
Yeah, I added this mainly to show the issue so we could discuss a possible solution such as adding `isFullWidth` to the `Dropdown`. That would have to be done separately though so I guess we actually may want to keep this and turn it into a warning. I'll hold off changing it for now until we can discuss more though.

Reply:
I went ahead and made this change locally, thanks.